### PR TITLE
Router events

### PR DESCRIPTION
**Pull request by samandmoore**
@dschaub

### CODE REVIEW

File: backbone.blazer.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ Backbone.Blazer.Router = Backbone.Router.extend({
         var router = this;
 
         route.trigger('before:execute', routeData);
+        router.trigger('before:execute', routeData);
```
Comment (dschaub) [inline backbone.blazer.js:61]: cool. since this is being fired on the router now, should routeData include the route name/pattern?

Comment (samandmoore) [inline backbone.blazer.js:61]: yea. probably.

Comment (samandmoore) [inline backbone.blazer.js:61]: i have to wire that stuff in to route data. probably can do that in a follow-up PR. working on something like that here: https://github.com/Betterment/backbone.blazer/compare/enhance-route-data

Comment (dschaub) [inline backbone.blazer.js:61]: right, cool :+1:

**Comment by dschaub** [comment]
lgtm

**Pull request merged**